backend-api/template.yaml

```diff
@@ -1373,4 +1373,8 @@ Outputs:
   ProxyApiUrl:
     Condition: ProxyApiDeployment
     Description: Proxy API Gateway Pretty DNS
-    Value: !Sub https://${ProxyApiDomainName}
\ No newline at end of file
+    Value: !Sub https://${ProxyApiDomainName}
+
+  PrivateApiUrl:
+    Description: Private API Gateway base URL
+    Value: !Sub https://${PrivateApi}.execute-api.${AWS::Region}.amazonaws.com/${Environment}/
```

Comment:
This has a `/` is this intentional? I don't think the proxyApiUrl has one so the tests might fail?